bootstrap/scss/_custom-styles.scss

```diff
@@ -83,6 +83,9 @@ footer {
     padding: 5rem; 
     background-color: #e9ecef;
     opacity: 0.7;
+    @media (max-width: 767px) {
+        padding: 1.25rem;
+    }
```

Comment:
We can use the media query mixins and variables from Bootstrap here.

Reply:
done